packages/core/workflows-sdk/src/utils/composer/create-workflow.ts

```diff
@@ -19,6 +19,16 @@ import {
 
 global[OrchestrationUtils.SymbolMedusaWorkflowComposerContext] = null
 
+/**
+ * Workflow response class encapsulates the return value of a workflow
+ */
+export class WorkflowResponse<TData, THooks = []> {
```

Comment:
suggestion: should we move that to its own file like the step response?

Reply:
Done